src/components/ContactForm/index.js

```diff
@@ -0,0 +1,170 @@
+import React, { useState } from "react";
+import Typography from "root/components/Typography";
+import Button from "root/components/Button";
+import classNames from "classnames";
+
+import "./index.css";
+
+function useForm() {
```

Comment:
I guess it's ok to leave this here since we are not using forms elsewhere, but it's good to point out that we usually have a hooks folder.